classes/class-twenty-twenty-one-custom-colors.php

```diff
@@ -160,26 +151,4 @@ public static function get_relative_luminance_from_hex( $hex ) {
 		$lum = ( 0.2126 * $red ) + ( 0.7152 * $green ) + ( 0.0722 * $blue );
 		return (int) round( $lum );
 	}
-
```

Comment:
This class is used to soften the color of the input field background so needs to be kept.

Reply:
:thinking: where? Ah, I assume it's on another PR that hasn't been merged yet? 'Cause there's nothing like that on `trunk`

Reply:
Added back :+1: